ProcessLib/HT/HTFEM.h

```diff
@@ -88,6 +88,73 @@ class HTFEM : public HTLocalAssemblerInterface
         return Eigen::Map<const Eigen::RowVectorXd>(N.data(), N.size());
     }
 
+    /// Computes the flux in the point \c pnt_local_coords that is given in
+    /// local coordinates using the values from \c local_x.
+    // TODO add time dependency
```

Comment:
Yep, that would be nice (for the sake of completeness). Could you please add it now?